File: j0eda21122aSeisSiete/C++/main.cpp
#include <iostream>
#include <stdlib.h>

#ifdef _WIN32
#define clear "cls"
#define pause "PAUSE"

#else
#define clear "clear"
#define pause "read -p '' var"

#endif

using std::cout;
using std::cin;

// g++ main.cpp && ./a.out && rm a.out

int main(){

    cout << "\n- Nickname: SeisSiete\n";
    cout << "- Carrera: Ingeniería en Computación\n";
    cout << "- Preparatoria: UTEG\n";
    cout << "- Lenguajes de Programación: C / C++ / Python\n\n";

    system(pause);
    system(clear);

    return 0;
}

File: j0eda21122aSeisSiete/Python/Main.py
import os
import time

if os.name in ["nt"]:
    clear = "cls"
    pause = "pause"
else:
    clear = "clear"
    pause = "read -p '' var"

# python3 j0/Python/Main.py

def main():
    print("- Nickname: SeisSiete")
    print("- Carrera: Ingeniería en Computación")
    print("- Preparatoria: UTEG")
    print("- Lenguajes de Programación: C / C++ / Python\n")

    os.system(pause)
    os.system(clear)

if __name__ == '__main__':
    main()